aftquery/aftsearch.py

```diff
@@ -0,0 +1,97 @@
+#coding=utf-8
+
+import re
+import sys
+import urllib
+import argparse
+from pprint import pprint
+from bs4 import BeautifulSoup as bs4
+
+from .parser.aft import clubs as aft_clubs
```

Comment:
I don't think this will work when you run the script from the command line. Did you test it?

Reply:
yeah aftsearch.py works fine on the refactoring that I did.